src/test/java/org/apache/commons/collections4/bloomfilter/DefaultBloomFilterTest.java

```diff
@@ -163,28 +123,30 @@ public boolean merge(BloomFilter other) {
                             String.format("Value in list %s is less than 0", indices.first()));
                 }
             }
-            return true;
         }
 
         @Override
-        public int cardinality() {
-            return indices.size();
+        public boolean merge(IndexProducer indexProducer) {
+            boolean result = indexProducer.forEachIndex((x) -> {
```

Comment:
You do not need parentheses around the `(x)` for single argument lambda functions.